ttnn/ttnn/__init__.py

```diff
@@ -466,6 +466,7 @@ def manage_config(name, value):
     logaddexp2_bw,
     squared_difference_bw,
     add_bw,
+    fmod_bw,
```

Comment:
Same as [this](https://github.com/tenstorrent/tt-metal/pull/9712/files#r1675360757)